features/minion_ssh_push.feature

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2016 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: register a salt-minion via SSH push
+
+  Scenario: bootstrap a sles minion
+     Given I am authorized
+     When I follow "Salt"
+     Then I should see a "Bootstrapping" text
+     And I follow "Bootstrapping"
+     Then I should see a "Bootstrap Minions" text
+     # sle-minion = sles, rh_minion = redhat
+     And  I enter the hostname of "sle-minion" as hostname
+     And I enter "22" as "port"
+     And I enter "root" as "user"
+     And I enter "linux" as "password"
+     And I click on "Bootstrap it"
+     Then take picture
```

Comment:
yeah, for moment, i put take a picture just for testing, I wlll put then  "i should see " or something like this, that take care of waiting that the minion is bootstrapped.